run.py

```diff
@@ -38,7 +38,7 @@ def get_args():
 options.binary_location = "/usr/bin/google-chrome-beta"
 options.add_argument("--no-sandbox")
 options.add_argument("--enable-quic")
-options.add_argument("--quic-version=h3-29")
+options.add_argument("--quic-version=80")
```

Comment:
Are you sure about that? It seems like the argument here is a string (it was `h3-29` before), not the integer from the enum.

Reply:
I believe it uses [this function](https://source.chromium.org/chromium/chromium/src/+/main:net/third_party/quiche/src/quic/core/quic_versions.cc;l=339;drc=404e8d654e8b26336d2cb103b9c21faecbf7f73a) for parsing the version string, which is pretty flexible (allows for version number, alpn, etc)

I would use alpn (h3), but it seems that might result in [Quic v2 being configured](https://source.chromium.org/chromium/chromium/src/+/main:net/third_party/quiche/src/quic/core/quic_versions.cc;drc=404e8d654e8b26336d2cb103b9c21faecbf7f73a;l=620)

Reply:
> I would use alpn (h3), but it seems that might result in [Quic v2 being configured](https://source.chromium.org/chromium/chromium/src/+/main:net/third_party/quiche/src/quic/core/quic_versions.cc;drc=404e8d654e8b26336d2cb103b9c21faecbf7f73a;l=620)

That functions goes in the opposite direction though, converting a parsed version to an ALPN. That doesn't mean that using the h3 identifier is not deterministic.
What happens if we just use `h3` here?

Reply:
The way I see it being used [here](https://source.chromium.org/chromium/chromium/src/+/main:net/third_party/quiche/src/quic/core/quic_versions.cc;drc=404e8d654e8b26336d2cb103b9c21faecbf7f73a;l=361), it looks like all Quic versions are iterated over, and the first that matches the given ALPN will be returned. QuicV2 shows up first in the [supported versions list](https://source.chromium.org/chromium/chromium/src/+/main:net/third_party/quiche/src/quic/core/quic_versions.h;drc=7aaf76be08204c728fb09fdd461d9f304b5cc2d7;l=411),  so it would get selected over QuicV1.

Reply:
Thanks for digging into this. Let's go with your suggestion then.